react/week2/to-do-app/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import './App.css';
+import ToDoList from './ToDoList';
+
+const todoList = [
+  {id: 1,description: "Get out of bed"},
+  {id: 2,description: "Brush teeth"},
+  {id: 3,description: "Eat breakfast"},
+];
+let counter=todoList.length+1;
+
+function App() {
+  const onButtonClick = () => {
+    let newToDo={};
+    newToDo.id=counter++;
+    newToDo.description='Random ToDo';
+    setTodos(todos.concat(newToDo));
+  }
+
+  const handleDelete = (id) => {
+    const newTodos = [...todos];
+    newTodos.splice(id-1, 1);
+    setTodos(newTodos);
+  }
+
+  const [todos, setTodos] = useState(todoList);
```

Comment:
In general it is good practice to move the different state variables to the top of the component. It makes it easier to get an overview of the component and it makes sure that no state variables are defined conditionally (react will complain about this if you had an if-statement above this line for example).

Reply:
Thank you, i have resolved this part of code